docs/guides/layers.rst

```diff
@@ -605,6 +585,126 @@ retiling a ``TiledRasterLayer``.
     # zoom_level will reamin the same in the resulting TiledRasterLayer
     tiled_raster_layer.tile_to_layout(layout=gps.GlobalLayout(zoom=11))
 
+
+Getting Point Values
+^^^^^^^^^^^^^^^^^^^^^
+
+The ``get_point_values`` takes a collection of ``shapely.geometry.Point``\s
+and returns the value(s) that are at the given point in the layer. The number
+of values returned depends on the amount of bands the values have. Each value
+corresponds to one layer.
```

Comment:
`Each value corresponds to one layer` unclear what this means. Are you using the word `layer` as stand-in for band?

Reply:
Ah, yeah I was. I'll fix that so it's more clear.